config/services.yaml

```diff
@@ -40,6 +44,10 @@ services:
             string $fromAddress: "%env(MAILER_FROM_ADDR)%"
             string $fromName: "%env(MAILER_FROM_NAME)%"
 
+    App\Util\MdappLogger:
+        arguments:
```

Comment:
Don't google what's the difference between Arguments and Bind

Reply:
I'll just use bind to be consistent.